### PR TITLE
Fontconfig update

### PR DESCRIPTION
**Pull request by alexeymorozov**
Closes https://github.com/fontist/fontist/issues/195

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -277,6 +278,25 @@ Roboto Mono:
     - "/Users/user/.fontist/fonts/RobotoMono-VariableFont_wght.ttf"
 ----
 
+=== Work with Fontconfig: `fontist fontconfig`
```
Comment (ronaldtse) [inline README.adoc:281]: @alexeymorozov can you help add more information about what fontconfig is, what platform it runs on, and how it is useful? Thanks.

Comment (alexeymorozov) [reply README.adoc:281]: Updated, should be better now.

**Comment by alexeymorozov** [comment]
@ronaldtse Updated metanorma specs. It's all green now. If anything, it's ready to go.

**Comment by ronaldtse** [comment]
Thanks @alexeymorozov !

**Pull request merged**